agent/consul/acl_server.go

```diff
@@ -95,6 +95,7 @@ func (s *Server) updateSerfTags(key, value string) {
 	s.updateEnterpriseSerfTags(key, value)
 }
 
+// TODO:
```

Comment:
Missing comment?

Reply:
This gets deleted in the followup PR

Reply:
Thanks for confirming! I thought it might , but didn't have a chance to look. I'll resolve this comment.